### DISCUSSION TITLE
fastddsgen -python option can't find template module.st [13638]

### DISCUSSION THREAD

**Discussion by webhoppery** [eProsima/Fast-DDS#2444] [Q&A]
Every time I use the fastddsgen -python option I get the following error :

ERROR<Exception>: Can't find template module.st; group hierarchy is [FastCdrCommon]
Exception in thread "main" java.lang.NullPointerException
        at com.eprosima.fastdds.fastddsgen.execute(fastddsgen.java:401)
        at com.eprosima.fastdds.fastddsgen.main(fastddsgen.java:1474)

I didn't see the module.st anywhere in the eProsima Fast DDS v2.5.0 installation.

I get the same error on both Ubuntu v20.04.3 and Win10 21H2 on same dual boot laptop. Both with Fast DDS v2.5.0.

Fastddsgen works fine on both installations without the -python option.

**Comment by JLBuenoLopez** [comment]
Hi @webhoppery,

I have opened an issue in the corresponding repository eProsima/Fast-DDS-Gen#88. We will continue the conversation there if you don't mind.
